include/libcyphal/types.hpp

```diff
@@ -189,7 +189,7 @@ struct PmrRawBytesDeleter final
 
         if ((nullptr != memory_resource_) && (nullptr != raw_bytes_ptr))
         {
-            CETL_DEBUG_ASSERT(isAligned(raw_bytes_ptr, size_bytes_), "Unexpected alignment of the memory buffer.");
+            // CETL_DEBUG_ASSERT(isAligned(raw_bytes_ptr, size_bytes_), "Unexpected alignment of the memory buffer.");
```

Comment:
Commenting usage of `isAligned` doesn't make it fail on CI... so, it seems that clang-tidy misses it for some reason